services/cd-service/pkg/service/overview.go

```diff
@@ -349,8 +351,48 @@ func (o *OverviewServiceServer) StreamOverview(in *api.GetOverviewRequest,
 	}
 }
 
+func (o *OverviewServiceServer) StreamChangedApps(in *api.GetChangedAppsRequest,
+	stream api.OverviewService_StreamChangedAppsServer) error {
+	ch, unsubscribe := o.subscribeChangedApps()
+	defer unsubscribe()
+	done := stream.Context().Done()
+	for {
+		select {
+		case <-o.Shutdown:
+			return nil
+		case changedAppsNames := <-ch:
+			if len(changedAppsNames) == 0 { //This only happens when a channel is first triggered, so we send all apps
```

Comment:
This ensures that the first message to some listener is always the full list of apps